snuba/datasets/processors/replays_processor.py

```diff
@@ -185,25 +185,76 @@ def process_message(
                 "project_id": message["project_id"],
             }
 
-            # # The following helper functions should be able to be applied in any order.
-            # # At time of writing, there are no reads of the values in the `processed`
-            # # dictionary to inform values in other functions.
-            # # Ideally we keep continue that rule
-            self._process_base_replay_event_values(processed, replay_event)
-            self._process_tags(processed, replay_event)
-            self._process_sdk(processed, replay_event)
-            self._process_kafka_metadata(metadata, processed)
-
-            # # the following operation modifies the event_dict and is therefore *not* order-independent
-            self._process_user(processed, replay_event)
-            self._process_event_hash(processed, replay_event)
-            self._process_contexts(processed, replay_event)
-            return InsertBatch([processed], None)
+            if replay_event["type"] == "replay_actions":
+                actions = process_replay_actions(replay_event, processed, metadata)
+                return InsertBatch(actions, None)
```

Comment:
@lynnagara Is there some internal limit to how many messages can be in an InsertBatch?

Reply:
do we limit the number of replay actions upstream at all in recordings consumer or sdk? cc @billyvg @cmanallen

Reply:
There are no limits on the SDK side

Reply:
i feel like we should put a bounds for bot cases, etc. something like 100 or 1000 per segment?